test/parallel/test-v8-serdes.js

```diff
@@ -63,7 +65,7 @@ const deserializerTypeError =
 {
   const ser = new v8.DefaultSerializer();
   ser._writeHostObject = common.mustCall((object) => {
-    assert.strictEqual(object, process.stdin._handle);
+    assert.strictEqual(object, hostObject);
     const buf = Buffer.from('stdin');
```

Comment:
[Curiosity / context question]: I assume that in the context of this test, `"stdin"` is meant to be just a random string constant. Would it be less confusing to change it to something like `"hostObjectTag"` or similar after this change? Or are you trying to keep the churn minimal?

Reply:
No, I just did search/replace, that’s all. ;) Thanks for catching is, updated!